Fix:

fatal error: 'malloc.h' file not found

malloc.h is Linux-specific so it should not be used in portable software.
POSIX says malloc is defined in stdlib.h which is already included.
--- dict/src/lib/rsa.cpp.orig
+++ dict/src/lib/rsa.cpp
@@ -3,7 +3,6 @@
 #include <stdlib.h>   
 #include <time.h>  
 #include <math.h>
-#include <malloc.h>
 
 #include "rsa.h"
 
